config/config.default.yaml

```diff
@@ -758,6 +757,14 @@ solving:
     linearized_unit_commitment: true
     horizon: 365
 
+  agg_p_nom_limits:
+    agg_offwind: false
+    include_existing: false
+    years:
+      2030: data/agg_p_nom_minmax.csv
+      2040: data/agg_p_nom_minmax.csv
+      2050: data/agg_p_nom_minmax.csv
```

Comment:
thanks for the contribution. I am but I see that the functionality is needed. perhaps we can add columns to the data file for different years instead?

Reply:
@FabianHofmann I've pushed an update.